inc/savedsearch.class.php

```diff
@@ -761,14 +761,20 @@ function displayMine() {
       $colspan = 2;
       echo "<table class='tab_cadre_fixehov'>";
       echo "<thead><tr><th colspan='$colspan' class='private_header'>" .
-                  sprintf(__('Private %1$s'), $this->getTypeName(count($searches['private']))) .
+                  sprintf(
+                     _n('Private %1$s', 'Privates %1$s', count($searches['private'])),
```

Comment:
no 's' in english.... but _n can be used with twice the same string, as for `_n('Software', 'Software', 1);`

Reply:
Indeed; thank you! :)